onadata/libs/utils/opendata_tools.py

```diff
@@ -0,0 +1,13 @@
+from onadata.libs.utils.common_tags import METADATA_FIELDS
+
+def remove_metadata_fields(data):
```

Comment:
can you please move this to logger_tools instead of a new file, that seems like a nice more generic location for it

Reply:
Sure. Moved that.